README.md

```diff
@@ -0,0 +1,23 @@
+# Cincinnati
+
+Cincinnati is a service to transform update information as modeled in a Docker registry (such as Quay.io or DockerHub) into a graph representation that can be interpreted by update clients. It is the spiritual successor to the Omaha protocol used to udate Google Chrome and Google Chrome OS based operating systems.
```

Comment:
Let's just use the copy from the docs.

"Cincinnati is an update protocol designed to facility automatic updates. It describes a particular method for representing transitions between releases of a project and allowing a client to perform automatic updates between these releases."